modelforge/potential/sake.py

```diff
@@ -125,20 +126,20 @@ def _model_specific_input_preparation(
 
         number_of_atoms = data.atomic_numbers.shape[0]
 
-        atomic_embedding = self.embedding(
-            F.one_hot(data.atomic_numbers.long(), num_classes=self.max_Z).to(
-                self.embedding.weight.dtype
-            )
-        )
+        # atomic_embedding = self.embedding(
```

Comment:
@ArnNag , I am a bit surprised about the embedding here. Why are you not directly passing `data.atomic_numbers`? What is the expected output?

Reply:
I commented out the original code for now, but I'd like to understand this